src/metadata/trackmetadatataglib.cpp

```diff
@@ -18,6 +18,12 @@
 #define TAGLIB_HAS_TAG_CHECK \
     (TAGLIB_MAJOR_VERSION > 1) || ((TAGLIB_MAJOR_VERSION == 1) && (TAGLIB_MINOR_VERSION >= 9))
 
+#ifdef _WIN32
+#define TAGLIB_FILENAME_FROM_QSTRING(fileName) (fileName).toStdWString().constData();
```

Comment:
I have just noticed that there is a typo. 
#define TAGLIB_FILENAME_FROM_QSTRING(fileName) (fileName).toStdWString().const();

We should fix it together with: 
https://github.com/mixxxdj/mixxx/pull/731#issuecomment-147366056